### ISSUE TITLE
Add options to format export lists

### ISSUE THREAD

**Issue by vrom911** [jaspervdj/stylish-haskell#220]
I guess it could be made in a similar manner as for imports: options for the padding, spaces after brackets or probably any better ideas.

**Comment by EncodePanda** [comment]
You mean like when module exports stuff

```
module Foo where
  ( bar
  , baz)
```

?

**Comment by vrom911** [comment]
Yes, exactly. I still think that this could be extremely useful :slightly_smiling_face: 

Some people prefer to align as you mentioned:

```haskell
module Foo
    ( bar
    , baz
    ) 
    where
```
while others would want to do something like this:

```haskell
module Foo where
       ( bar
       , baz
       ) where
```

It would be nice to have this automatically and configurable.

Some concerns I can see are related to `Main` module. usually you do:

```haskell
module Main (main) where
```

but not

```haskell
module Main 
    ( main
    ) where
```

However, I am not sure if this should be handled separately.

**Comment by EncodePanda** [comment]
considered it done

**Comment by EncodePanda** [comment]
hey @vrom911 I'm happy to report that this feature was added few versions ago and was additionally polished recently here https://github.com/haskell/stylish-haskell/pull/357 (not yet released). Please have a look and close the ticket if you think that this is what you were hoping to get.
